stablehlo/dialect/Version.cpp

```diff
@@ -49,6 +49,8 @@ static FailureOr<std::array<int64_t, 3>> extractVersionNumbers(
 }  // namespace
 
 FailureOr<Version> Version::fromString(llvm::StringRef versionRef) {
+  if (versionRef == "current") return Version::getCurrentVersion();
+  if (versionRef == "minimum") return Version::getMinimumVersion();
```

Comment:
Agree. But making as a part of this change to avoid the temporary redundancy.

There is some oddity that this aliasing is now needed in two places:
1. Calls to serialize via `stablehlo-translate --serialize --target=current`
2. Explicit pass option to `--vhlo-to-version='target=current'`

Making this change now avoids the redundancy.

Reply:
Actually.. we don't need to support aliases in VhloToVersion.. Can make sure call sites use explicit version numbers and bump this logic into `Serialization.cpp`.

Reply:
Moved this logic to `Serialization.cpp` and only accept fully qualified version numbers in `VhloToVersion`.